src/main/java/duke/IllegalChatBotListArgumentException.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+/**
+ * Exception when a wrong task query string is supplied to ChatBotList.
+ */
+public class IllegalChatBotListArgumentException extends ChatBotListException {
+    public IllegalChatBotListArgumentException(String message) {
+        super(message);
+    }
+    public IllegalChatBotListArgumentException() {
```

Comment:
Should there be an empty line before this line? Also noticed in a few other files